content/app/models/file-path.js

```diff
@@ -0,0 +1,90 @@
+define("content/app/models/file-path", function(require, exports, module) {
+
+	/**
+	 * Simple class used for handling of file paths.
+	 */
+	var FilePath = Class({
+
+		$statics : {
+			FOLDER_SEPARATOR : '/',
```

Comment:
I'd rather use regular expression: /(\/|\\)/g or /[\/\\]{1}/g
That will cover both \ and / separators.